OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -452,7 +452,7 @@ Action InitInputPanel(Widget panel)
 				};
 
 				var header = ScrollItemWidget.Setup(hotkeyHeader, returnTrue, doNothing);
-				header.Get<LabelWidget>("LABEL").GetText = () => "Game Commands";
+				header.Get<LabelWidget>("LABEL").GetText = () => "Game commands";
```

Comment:
As an aside, having "Commands" after everything is silly.  When the input panel was first set up this made sense, but the additions since then have blurred the line (many of the things under "Game Commands" aren't commands, and none of the music keys are commands).  At some point in the future (__not this pr__) it would be worth regrouping the hotkeys into more appropriately named sections.